src/redux/books/books.js

```diff
@@ -0,0 +1,22 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
```

Comment:
- [ ]  You have well configured the Redux Store and write actions and reducer for adding and removing books perfectly however I'm getting this error message in the console when I'm trying to add or remove a book, Kindly fix this and make sure the user can add and a new book remove it from the 
list without any error 😜 👍🏻 

![console](https://user-images.githubusercontent.com/79058364/173524551-59888ac4-73ed-495f-b3d8-ed00300d600a.png)

Reply:
- Hi @Fatima-hub333, have you been able to solve the problem since you didn't reply to my message on slack?